src/main/java/org/apache/commons/imaging/formats/tiff/datareaders/DataReaderTiled.java

```diff
@@ -125,8 +125,8 @@ private void interpretTile(final ImageBuilder imageBuilder, final byte[] bytes,
         // verify that all samples are one byte in size
         final boolean allSamplesAreOneByte = isHomogenous(8);
 
-        if (predictor != 2 && bitsPerPixel == 24 && allSamplesAreOneByte) {
-            int k = 0;
+          if (bitsPerPixel == 24 && allSamplesAreOneByte
```

Comment:
I think this `if` is not aligned with the `final` statement above. Haven't looked at the code in the IDE, but maybe it is formatted with wrong number of spaces?

Reply:
The alignment is now fixed.